Tests/conf.json

```diff
@@ -165,6 +165,9 @@
         },
         {
             "playbookID": "endpoint_enrichment_-_generic_test"
+        },
+        {
+            "playbookID": "IOC Hunt - TIE - Test"
```

Comment:
You need to add which integration playbook will use `McAfee Threat Intelligence Exchange`